dev/ci/periodics/e2e-service-bigquery

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd ${REPO_ROOT}
+
+export ONLY_TEST_APIGROUP=bigquery.cnrm.cloud.google.com
```

Comment:
Does this flag accept a list? KCC have 3 services related to BigQuery 
`bigqueryconnection.cnrm.cloud.google.com/v1alpha1` (SciF, v1.123, broken, needs https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2742)
`bigquery.cnrm.cloud.google.com/v1beta1` (TF-based, in migration)
`bigquerydatatransfer.cnrm.cloud.google.com/v1alpha1` (SciFi, v1.123)

Reply:
Hmmm, it could :-)